guiEditor/src/components/propertyTab/propertyGrids/gui/commonControlPropertyGridComponent.tsx

```diff
@@ -11,6 +11,23 @@ import { TextInputLineComponent } from "../../../../sharedUiComponents/lines/tex
 import { LockObject } from "../../../../sharedUiComponents/tabs/propertyGrids/lockObject";
 import { OptionsLineComponent } from "../../../../sharedUiComponents/lines/optionsLineComponent";
 
+const sizeIcon: string = require("../../../../sharedUiComponents/imgs/sizeIcon.svg");

```

Comment:
When you get the chance please add a todo (for "Future") to move those to imports instead of requires :-)